Sources/ArgumentParser/Utilities/SequenceExtensions.swift

```diff
@@ -9,14 +9,9 @@
 //
 //===----------------------------------------------------------------------===//
 
-extension Sequence where Element: Equatable {
-  func uniquified() -> [Element] {
-    var sequence = Array<Element>()
-    for element in self {
-      if !sequence.contains(element) {
-        sequence.append(element)
-      }
-    }
-    return sequence
+extension Sequence where Element: Hashable {
+  func uniquified() -> [Iterator.Element] {
```

Comment:
This one too, please!
```suggestion
  func uniquified() -> [Element] {
```